k8s-scripts.config

```diff
@@ -41,3 +41,5 @@ JOBS=()
 # List of files ending in '.blockingjob.yml' in the kube directory
 BLOCKING_JOBS=()
 
+# List of files ending in '.cronjob.yml' in the kube directory
+CRONJOBS=()
```

Comment:
Was there a reason for not having a newline at the end of this file? It seems like it would help for anyone who `cat`s the file to have the usual newline there.